modules/vba_documents/lib/vba_documents/document_request_validator.rb

```diff
@@ -7,7 +7,7 @@ class DocumentRequestValidator
     include PDFUtilities
 
     SUPPORTED_CONTENT_TYPES = %w[application/pdf].freeze
-    MAX_FILE_SIZE_IN_BYTES = 100_000_000 # 100 MB
+    MAX_FILE_SIZE_IN_BYTES = 100_048_576 # 100 MB
```

Comment:
It doesn't look like this change alone is sufficient. I'm looking at L16 below where we divide this number by `1_000_000` to get the number of megabytes to use for the error message. We want that error message to continue to be "Document exceeds the file size limit of 100 MB," and changing this number alone would cause it be "Document exceeds the file size limit of 100.048576... MB."

Reply:
Good catch, thanks

Reply:
Ok, i fixed it.  We actually don't end up using that error message, we override it over in PDFUtilities#formatted_file_size